package.json

```diff
@@ -14,8 +14,8 @@
   "scripts": {
     "build": "tsc",
     "build:fixtures": "npm run build && node lib/tools/update-fixtures",
-    "test:full": "npm run build && mocha $(find lib/test -name *_test.js) && npm run lint",
-    "test:fast": "npm run build && mocha $(find lib/test/unit -name *_test.js)",
+    "test:full": "npm run build && bash test/run-mocha.sh && npm run lint",
```

Comment:
I use bash on my machine so I can't test that this breaks on others, but change to `./test/run-mocha.sh` to support non-bash environments?

Reply:
I can change it to that. I was basing it off of lighthouse which is what I used as reference to getting mocha to run. https://github.com/GoogleChrome/lighthouse/blob/master/package.json#L32

Reply:
```
stramel@DESKTOP-6G41VHH MINGW64 ~/Projects/GitHub/polymer-modulizer (ms/test-windows)
$ npm test

> polymer-modulizer@0.3.0 test C:\Users\stramel\Projects\GitHub\polymer-modulizer
> npm run test:fast && npm run lint


> polymer-modulizer@0.3.0 test:fast C:\Users\stramel\Projects\GitHub\polymer-modulizer
> npm run build && ./test/run-mocha.sh unit


> polymer-modulizer@0.3.0 build C:\Users\stramel\Projects\GitHub\polymer-modulizer
> tsc

'.' is not recognized as an internal or external command,
```

Reply:
huh, I thought npm handled that on windows. Okay, bash is fine for now, if anyone is on a shell that can't run that we can cross that bridge when we come to it.